internal/controller/finbackupconfig_controller.go

```diff
@@ -41,9 +59,41 @@ func NewFinBackupConfigReconciler(
 // For more details, check Reconcile and its Result here:
 // - https://pkg.go.dev/sigs.k8s.io/controller-runtime@v0.16.3/pkg/reconcile
 func (r *FinBackupConfigReconciler) Reconcile(ctx context.Context, req ctrl.Request) (ctrl.Result, error) {
-	_ = log.FromContext(ctx)
+	logger := log.FromContext(ctx)
+
+	var fbc finv1.FinBackupConfig
+	if err := r.Get(ctx, req.NamespacedName, &fbc); err != nil {
+		return ctrl.Result{}, client.IgnoreNotFound(err)
+	}
+
+	var pvc corev1.PersistentVolumeClaim
+	if err := r.Get(ctx, types.NamespacedName{Namespace: fbc.Namespace, Name: fbc.Spec.PVC}, &pvc); err != nil {
+		return ctrl.Result{}, fmt.Errorf("failed to get PVC %s/%s: %w", fbc.Namespace, fbc.Spec.PVC, err)
+	}
+
+	clusterID, err := getCephClusterIDFromPVC(ctx, r.Client, &pvc)
+	if err != nil {
+		return ctrl.Result{}, fmt.Errorf("failed to get Ceph cluster ID: %s: %s: %w", fbc.Namespace, fbc.Spec.PVC, err)
+	}
+	if clusterID != r.finCephClusterID {
+		logger.Info("the target pvc is not managed by this controller")
+		return ctrl.Result{}, nil
+	}
```

Comment:
We've already had a utility function for this process https://github.com/cybozu-go/fin/blob/358835532285e8c5244b870ace85b5215b359df7/internal/controller/util.go#L107-L128, so let's use this.